File: VDFEditor/src/main/java/de/treichels/hott/vdfeditor/ui/Language.java
package de.treichels.hott.vdfeditor.ui;

import java.util.ResourceBundle;

public enum Language {
    DE, EN, FR, IT, ES, NL, CZ, RU, CN, KR, JP;

    /** @return the locale-dependent languages */
    @Override
    public String toString() {
        return ResourceBundle.getBundle(getClass().getName()).getString(name());
    }
}

File: VDFEditor/src/main/kotlin/de/treichels/hott/vdfeditor/ui/Language.kt
package de.treichels.hott.vdfeditor.ui

import java.util.*

enum class Language {
    DE, EN, FR, IT, ES, NL, CZ, RU, CN, KR, JP;

    /** @return the locale-dependent languages
     */
    override fun toString(): String = ResourceBundle.getBundle(javaClass.name).getString(name)
}
